config/lib/express.js

```diff
@@ -112,10 +112,10 @@ module.exports.initSession = function (app, db) {
 		saveUninitialized: true,
 		resave: true,
 		secret: config.sessionSecret,
-		store: new MongoStore({
-			db: db.connection.db,
-			collection: config.sessionCollection
-		})
+        store: new MongoStore({
```

Comment:
@ilanbiala I can see that in come files tabs are used for indentation and in come files spaces. What is currently preferred?